### PR TITLE
Add bancontact specs

### PR DESCRIPTION
**Pull request by daniel-munch-cko**

### CODE REVIEW

File: scripts/sync-generated-specs.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ var syncPaymentSources = function()
   syncPaymentResponse('Ideal', 'http://sb-gateway-internal.cko.lon/ideal-internal-api/relations/gw/payment', '05');
   syncPaymentRequest('Klarna', 'http://sb-gateway-internal.cko.lon/klarna-internal/relations/gw/pay', '10');
   syncPaymentResponse('Klarna', 'http://sb-gateway-internal.cko.lon/klarna-internal/relations/gw/payment', '06');
+  syncPaymentResponse('Bancontact', 'http://sb-gateway-internal.cko.lon/ppro-internal/bancontact/relations/gw/pay', '15');
```
Comment (philippe-leonhardt-cko) [inline scripts/sync-generated-specs.js]: Is 2x `syncPaymentResponse` correct?

Comment (kevin-reece-cko) [reply scripts/sync-generated-specs.js]: The fix for this seems to be in PR 44

File: spec/components/schemas/Payments/PaymentRequestSource.yaml [hunk 2]
```diff
@@ -17,6 +17,7 @@ discriminator:
     qiwi: '#/components/schemas/12_PaymentRequestQiwiSource'
     safetypay: '#/components/schemas/13_PaymentRequestSafetyPaySource'
     sofort: '#/components/schemas/14_PaymentRequestSofortSource'
+    bancontact: '#/components/schemas/15_PaymentRequestBancontactSource'
```
Comment (kevin-reece-cko) [inline spec/components/schemas/Payments/PaymentRequestSource.yaml]: This should be number "07" (and the actual filename too). 

The non-APM payment methods come first, and then the APM payment methods follow alphabetically. 
This is for the filename numerical prefix (which affects actual dropdown ordering), and the ordering in this file (for source readability)

Comment (kevin-reece-cko) [reply spec/components/schemas/Payments/PaymentRequestSource.yaml]: Note: the following filename numbers all have to be incremented too. Quite annoying really

Can you also add a comment explaining this please :)

File: spec/components/schemas/Payments/PaymentResponseSource.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ discriminator:
     qiwi: '#/components/schemas/08_PaymentResponseQiwiSource'
     safetypay: '#/components/schemas/09_PaymentResponseSafetyPaySource'
     sofort: '#/components/schemas/10_PaymentResponseSofortSource'
-
+    bancontact: '#/components/schemas/15_PaymentResponseBancontactSource'
```
Comment (kevin-reece-cko) [inline spec/components/schemas/Payments/PaymentResponseSource.yaml]: This should be number "03" (and the actual filename too). 

(See comments on PaymentRequestSource.yaml above)

File: scripts/sync-generated-specs.js (hunk 1 above)
Comment (philippe-leonhardt-cko) [reply scripts/sync-generated-specs.js]: Ah yes, I saw it.

**Pull request merged**